calibrations/sepd/eventplane/scripts/genFun4All.sh

```
@@ -0,0 +1,43 @@
+nEvents=${5}
+dbtag=${6}
+submitDir=${7}
+
+# extract runnumber from file name
+file=$(basename "$input")
+IFS='-' read -r p1 p2 p3 <<< "$file"
+run=$(echo "$p2" | sed 's/^0*//') # Remove leading zeros using sed
+
+if [[ ! -z "$_CONDOR_SCRATCH_DIR" && -d $_CONDOR_SCRATCH_DIR ]]
+then
+    cd $_CONDOR_SCRATCH_DIR
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)